src/main/java/duke/Helper.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+/**
+ * Contains helper functions that are used across several classes.
+ */
+public class Helper {
+    /**
+     * Joins a sub-array of strings into 1 string where each element in sub-array is separate by a space.
+     *
+     * @param arr array containing sub-array to join
+     * @param start start index of sub-array to join
+     * @param end end index of sub-array to join
+     * @return string of sub-array joined with space
+     */
+    public static String join(String[] arr, int start, int end) {
```

Comment:
I think there should be punctuations behind the parameter description? As per the Java Coding Standards.